src/documents/MeetLearnerNeedsAndPreferences.md

```diff
@@ -43,34 +55,52 @@ Examples of supplementation include:
 * Providing descriptions for photos, figures, diagrams, and other images
 
 <figure>
-[![Screenshot of Video player with transcript on](/assets/images/thumbs/400px-Videoplayer-captions-transcripts.jpg)](/assets/images/thumbs/400px-Videoplayer-captions-transcripts.jpg)
+[![Screenshot of Video player with transcript
+on](/assets/images/thumbs/400px-Videoplayer-captions-transcripts.jpg)](/assets/images/thumbs/400px-Videoplayer-captions-transcripts.jpg)
 <figcaption>
 *Figure 2:* A screen of the Floe Video player with transcripts displayed
 </figcaption>
 </figure>
 
-*Recreation:* Not all learning resources can be naturally tweaked or supplemented to provide good learning experiences. Some learning resources are in a medium where no natural translation exists. For instance, there is no easy way to translate a symphony orchestra performance into a non-aural experience. In these cases, a revisit of the content’s intent and meaning is needed to provide an interpretation of the content into a different medium.
+*Recreation:* Not all learning resources can be naturally tweaked or supplemented to provide good learning experiences.
+Some learning resources are in a medium where no natural translation exists. For instance, there is no easy way to
+translate a symphony orchestra performance into a non-aural experience. In these cases, a revisit of the content’s
+intent and meaning is needed to provide an interpretation of the content into a different medium.
 
 Examples of recreation include:
 
-* For a graphic novel, provide a written novel “equivalent”, based on an interpretation of a combination of the visuals and dialogue.
+* For a graphic novel, provide a written novel “equivalent”, based on an interpretation of a combination of the visuals
+  and dialogue.
 
 <a name="TheImportanceOfNaturalLanguage"></a>
```

Comment:
Named anchors are having the `link-external` class applied.

![image](https://user-images.githubusercontent.com/564762/98135962-6db99400-1e8e-11eb-9cbb-ed11b3e64c5e.png)

Reply:
@jhung Can you open a ticket for this? It'll need to be addressed in guide.inclusivedesign.ca as well. Thanks.

Reply:
Resolved in 336a17d.